app/scripts/filters/resources.js

```diff
@@ -349,18 +349,22 @@ angular.module('openshiftConsole')
         return url;
     };
   })
-  .filter('routeLabel', function(routeHostFilter, routeWebURLFilter, isWebRouteFilter) {
+  .filter('routeWebURLNoPath', function(routeHostFilter){
```

Comment:
instead of making a new function, i would change routeWebURL filter to take in an optional parameter that tells it not to include the path